tc/sgx/trusted_worker_manager/enclave/CMakeLists.txt

```diff
@@ -15,122 +15,16 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.2 FATAL_ERROR)
 PROJECT(libtcf-enclave  CXX)
 
-LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
-
-SET (EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR}/bin)
-SET (LIBRARY_OUTPUT_PATH ${CMAKE_BINARY_DIR}/lib)
-
-SET(DEPS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/deps")
-SET(TCF_TOP_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../../../")
-SET(TCF_CORE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../")
```

Comment:
TCF_TOP_DIR  and TCF_CORE_DIR are relative paths and may not be common across enclaves. Please check.  If not, you can keep this in CMakeLists.txt

Reply:
Yes, the core Dir may differ for sub directories. TCF_TOP_DIR and TCF_CORE_DIR is moved back to CMakeLists.txt.